app/services/curation_concern/generic_work_actor.rb

```diff
@@ -123,6 +123,7 @@ def attach_file(file)
       )
       generic_file.embargo_release_date = curation_concern.embargo_release_date
       generic_file.visibility = visibility
+      generic_file.owner = user.user_key
```

Comment:
The file owner isn't the current user, in the case of delegate submitted works.

Compare with https://github.com/uclibs/curate/blob/develop/app/services/curation_concern/base_actor.rb#L77